### ISSUE TITLE
KeyError: 'name'

### ISSUE THREAD

**Issue by cvergari** [sphinx-contrib/restbuilder#16]
Hi,

I have a reproducible bug occurring when processing a Python file with a class docstring mentioning another class in the same file. 

Here is an example Python file:

```
class class_one():
    """docstring of class_one. """


class class_two():
    """docstring of class two.

        Attributes:
            some_attribute (class_one): path to dicom file. Can be set directly.

    """

    some_attribute = class_one()
```

And my `index.rst`:

```
class_one
----------------

.. autoclass:: sphinx_bug.code.classes.class_one
    :members:
	
	
class_two
----------------

.. autoclass:: sphinx_bug.code.classes.class_two
    :members:
```

When I launch `make rst` I get:
``` 
Exception occurred:
  File "C:\Users\Claudio\miniconda3\envs\imaging\lib\site-packages\docutils\nodes.py", line 625, in __getitem__
    return self.attributes[key]
KeyError: 'name'
The full traceback has been saved in C:\Users\Claudio\AppData\Local\Temp\sphinx-err-npr86koc.log, if you want to report the issue to the developers.
```

I think this is a restbuilder issue because `make html` works (and `class_one` is correctly linked). The RST output works if I substitute `some_attribute (class_one):` with `some_attribute (int):`.
 
I am using: 
- sphinxcontrib-restbuilder 0.2 (from conda)
- Sphinx version: 3.4.3
- Python version: 3.6.12 (CPython)
- Docutils version: 0.16 release
- Jinja2 version: 2.11.2

I am attaching the full log: [sphinx-err-r16s30r_.log](https://github.com/sphinx-contrib/restbuilder/files/5904874/sphinx-err-r16s30r_.log)
Thank you!

**Comment by macfreek** [comment]
Hi @cvergari,

Thanks for reporting this!

You are correct, this is a bug in sphinx-restbuilder. It was fixed in 41b4d81 some time ago, but this fix has not made it to a release. I will create a new release (0.3) later this month.

In the mean time, as a workaround, you can manually install the latest version using:

> git clone https://github.com/sphinx-contrib/restbuilder.git
> cd restbuilder
> sudo python setup.py install

Let me know if this fixed the problem for you.

**Comment by cvergari** [comment]
Yes, this fixed the bug. Thanks! Looking forward for the next release.
